src/Proto.Actor/LocalContext.cs

```diff
@@ -31,7 +33,7 @@ public class Context : IMessageInvoker, IContext, ISupervisor
         private bool _stopping;
         private HashSet<PID> _watchers;
         private HashSet<PID> _watching;
-        private readonly ILogger logger = Log.CreateLogger<Context>();
+        private readonly ILogger _logger = Log.CreateLogger<Context>();

```

Comment:
Can we share the logger between contexts?
The current design also comes with a weight per actor.